frontend/src/lib/lemon-ui/LemonSegmentedSelect/LemonSegmentedSelect.tsx

```diff
@@ -17,7 +17,7 @@ export function LemonSegmentedSelect<T extends React.Key>({
         windowSize: { width = 0 },
     } = useWindowSize()
 
-    if (props.options.length >= shrinkOn || width < props.options.length * 100) {
+    if ((shrinkOn != null && props.options.length >= shrinkOn) || width < props.options.length * 100) {
```

Comment:
Is this related to the snapshot changes?

Reply:
Nope, those will be fixed once I rebase this PR - fixed it in the underlying branch. This is here to allow us to not pass `shrinkOn` to this component in case we don't want it to ever shrink (like the web analytics/web vitals button)